apps/fishing-map/features/reports/title/ReportTitle.tsx

```diff
@@ -19,25 +30,73 @@ type ReportTitleProps = {
 
 export default function ReportTitle({ area }: ReportTitleProps) {
   const { t } = useTranslation()
+  const { dispatchQueryParams } = useLocationConnect()
+  const dispatch = useAppDispatch()
   const areaDataview = useSelector(selectReportAreaDataview)
   const report = useSelector(selectCurrentReport)
+  const urlBufferValue = useSelector(selectUrlBufferValueQuery)
+  const urlBufferUnit = useSelector(selectUrlBufferUnitQuery)
+  const fitBounds = useMapFitBounds()
+  const { cleanFeatureState } = useFeatureState(useMapInstance())
+
+  const [bufferValue, setBufferValue] = useState<number>(urlBufferValue || DEFAULT_BUFFER_VALUE)
```

Comment:
Is this state needed for performance? Or could we use directly the previewBuffer in redux?

Reply:
🔩